rasa/cli/data.py

```diff
@@ -498,5 +539,23 @@ def _get_rules_path(path: Text) -> Path:
         rasa.shared.utils.cli.print_error_and_exit(
             f"'{rules_file}' needs to be the path to a file."
         )
+    elif rules_file.is_file():
+        rasa.shared.utils.cli.print_info(
+            f"Output file '{rules_file}' did not exist and will be created."
+        )
+        rasa.shared.utils.io.create_directory_for_file(rules_file)
 
     return rules_file
+
+
+def _dump_rules(path: Path, new_rules: List[StoryStep]) -> None:
+    if not new_rules:
+        return
+
+    existing_rules = []
+    if path.exists():
```

Comment:
the user should be alerted when rules are appended to an existing file IMO

Reply:
is the dumped output being tested yet? In addition I think we should test the migrated rules to an existing rules file

Reply:
yes, it's already tested: `test_convert_config_with_output_file_containing_data`. Will add the print 👍